cpp/utility.cc

```diff
@@ -87,4 +88,14 @@ std::shared_ptr<grpc::ServerCredentials> GetServerCredentials(
   assert(0);
 }
 
+std::unique_ptr<grpc::Server> StartAdminServer(const std::string& port) {
+  std::string server_address = "localhost" + port;
+  grpc::EnableDefaultHealthCheckService(true);
+  grpc::ServerBuilder builder;
+  builder.AddListeningPort(server_address, grpc::InsecureServerCredentials());
+  grpc::AddAdminServices(&builder);
+  return builder.BuildAndStart();
+  std::cout << "Admin Server listening on " << server_address << std::endl;
```

Comment:
The Java example emits the message "Admin & health server started, listening on...". How about adding " & health" after "Admin" in this message?

Reply:
Done